R/tribble.R

```diff
@@ -9,6 +9,17 @@
 #'
 #' @param ... Arguments specifying the structure of a `tibble`.
 #'   Variable names should be formulas, and may only appear before the data.
+#'
+#'   If a function needs to be applied to the data for a column (e.g. `factor`),
+#'   this should apear after the `~`, with the column name before,
+#'   e.g. `colA~factor(.)`. `.` should be used as a placeholder for the data.
+#'   Other arguments can be provided as needed, e.g.
+#'   `colA~factor(., levels = C("A", "B")`.
```

Comment:
Can we perhaps support the syntax `colA = factor(levels = ...)` instead, and `colA =` for untyped columns? We should still support, but deprecate, the old `~colA` syntax.